apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/index.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import domReady from '@wordpress/dom-ready';
+import { dispatch } from '@wordpress/data';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import 'a8c-fse-common-data-stores';
+import '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import { inIframe } from '../../../block-inserter-modifications/contextual-tips/utils';
+import { STEP_BY_STEP_LAUNCH_FLOW, FOCUSED_LAUNCH_FLOW, REDIRECT_LAUNCH_FLOW } from '../constants';
+import './styles.scss';
+
+let handled = false;
+
+const launchFlowTracksSlug: Record< string, string > = {
+	[ STEP_BY_STEP_LAUNCH_FLOW ]: 'gutenboarding-launch',
+	[ FOCUSED_LAUNCH_FLOW ]: 'focused-launch',
+	[ REDIRECT_LAUNCH_FLOW ]: 'launch-site',
+};
```

Comment:
Was wondering if you wanna streamline this or leave as is? Basically it looks like this atm:
```
{
"step-by-step": "gutenboarding-launch", 
"focused": "focused-launch",
"redirect": "launch-site"
}
```

Reply:
Ideally, we could initialize the `window.wpcomEditorSiteLaunch.launchFlow` property directly with the values in this object instead of keys so we don't need a map anymore.

Reply:
Done. https://github.com/Automattic/wp-calypso/pull/49004/commits/30af0a4f1899c82679f2b3ca59c7123f5731fc43